hooks/ruby/delete-repository-event/app.rb

```diff
@@ -0,0 +1,61 @@
+# Hook example for notifying an administrator in a repository by creating an issue when a repository is deleted.
+#
+# Needs the following environment variables
+#   GITHUB_HOST - the domain of the GitHub Enterprise instance. e.g. github.example.com
+#   GITHUB_API_TOKEN - a Personal Access Token that has the ability to create an issue in the notification repository.
+#   GITHUB_NOTIFICATION_REPOSITORY - the repository in which to create the nofication issue. e.g.
+#
+# Dependencies:
+#   octokit   - https://github.com/octokit/octokit.rb
+#   sinatrarb - http://www.sinatrarb.com/
+
+require 'octokit'
+require 'sinatra'
+require 'json'
+
+enable :logging
+github_api_token               = ENV['GITHUB_API_TOKEN']
+github_notification_repository = ENV['GITHUB_NOTIFICATION_REPOSITORY']
+github_host_url                = ENV['GITHUB_HOST']
```

Comment:
It should probably be something other than `github_host_url` because it's supposed to be the FQDN and not the URL.